modules/adding-helm-repository.adoc

```diff
@@ -15,14 +20,16 @@
 $ helm repo add rhacs https://mirror.openshift.com/pub/rhacs/charts/
 ----
 +
-The Helm repository for {product-title} includes two Helm charts for installing different components.
+The Helm repository for {product-title} includes Helm charts for installing different components.
 
+ifndef::cloud-svc[]
 ** Central services Helm chart (`central-services`) for installing the centralized components (Central and Scanner).
 +
 [NOTE]
 ====
 You deploy centralized components only once and you can monitor multiple separate clusters by using the same installation.
 ====
+endif::[]
 ** Secured Cluster Services Helm chart (`secured-cluster-services`) for installing the per-cluster (Sensor and Admission controller) and per-node (Collector) components.
```

Comment:
This list looks odd in the cloud-svc version. It is hard to tell that it is a list and it looks like a sub-step with an incomplete sentence; maybe an _including:_  at the end of Line 24?